README.md

```diff
@@ -1,2 +1,34 @@
-## Dev-Mountain Solidity Contract
+# **Dev. Mountain Tech Festival(Thailand) 2022 - Solidity Contract**
 
+
+## What is Dev. Mountain Tech Festival  
+
+Dev. Mountain Tech Festival is Thailand tech events that setup at
+18-20 March 2022 Toscana Valley Khao Yai.This event is have many top-tier Thai developers as speaker and have many different topic for developers or anyone who interested in technology
+in additional, this tech event also has blockchain hackathon inside it
+this Hackathon is having 5 team that passing the first round before can participate in the main event .
+[Facebook page](https://www.facebook.com/devmountaintechfestival/?ref=page_internal)
+## About this repo
+This repository is for the Solidity Contract (ERC721) <br>
+:bookmark_tabs: -> [Contract address](https://testnet.bscscan.com/address/0x4b5e5d9fe104722e75b9b671a20efa31c9d2bd75#code) <br>
+:newspaper: ->        [Ours Presentation (Thai)](https://docs.google.com/presentation/d/16d5bD5Ru74sfditDaeJPu73X5vpyMydQS6dL8hUu3EQ/edit#slide=id.g11e4d075bea_1_27)
+### Ours Hackathon problem that we got
+**Certificate verification for education** <br> 
+The main goal is want to removes suspicion about the certificatecontent and the issuer educational institution.
```

Comment:
- ตรง certificatecontent มันเป็นคำเดียวกันเลยหรือเป็น certificate content อันนี้สงสัย พอดี ไม่คุ้นคำนี้
- The main goal is want to removes ... ตรงนี้เราใช้ to เพื่ออธิบายจุดประสงค์ปกติจะใช้ to-infinitive กันคิดว่าไม่ต้องเติม s ที่ remove ใช้เป็น The main goal is want to remove ... ไปเลย